client/app/telecom/pack/xdsl/pack-xdsl.controller.js

```diff
@@ -6,6 +6,7 @@ angular.module('managerApp').controller('PackXdslCtrl',
     const noModemStatus = 404;
     const self = this;
     const availableModemTabStatus = ['active', 'migration', 'upgradeOffer'];
+    const SDSL = 'sdsl';
```

Comment:
Just a suggestion : You could have this constant in a separate file (at client/app/telecom/pack/xdsl level), as to allow this constant to be used elsewhere, and not to have hidden constant in controller files.
Like this you can even set availableModemTabStatus, in the constant file

Reply:
Yes, good remark. I did it.